test/runtests.jl

```diff
@@ -50,7 +50,9 @@ if anyerrors
     throw("Tests failed")
 end
 
-stdin = joinpath(dirname(@__FILE__), "stdin.sh")
-ENV2 = copy(ENV)
-ENV2["JULIA_HOME"] = JULIA_HOME
-run(setenv(`bash $stdin`, ENV2))
+if !is_windows()
```

Comment:
To support 0.6 here, we need the (forthcoming) new version of DataArrays that supports 0.6 as the minimum DataArrays version requirement for the package. Since that's 0.6-only, this will similarly have to be made 0.6-only. Right?

Older versions of the package should be upper bounded at Julia 0.6 as a precaution, since the current state of affairs is known not to work on 0.6 due to DataArrays issues. That's what we had talked about with Tony in triage.